### PR TITLE
User ratings

### PR DESCRIPTION
**Pull request by Jchew1234**
I created the user ratings model, controller, updated the routes and also tried out a simple form in the view.

### CODE REVIEW

**Review by a-jennings — APPROVED** [review]
Fixed?

File: app/controllers/user_ratings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class UserRatingsController < ApplicationController
+  skip_before_action :authenticate_user!, only: %i[new]
+  def new
+    @user_rating = UserRating.new
+    @user = User.find(params[:user_id])
```
Comment (louisizumi) [inline app/controllers/user_ratings_controller.rb]:
```suggestion
    @users = User.where('session_id == current_user.session_id')
```

File: app/controllers/user_ratings_controller.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    # raise
+  end
+
+  def create
+    @user_rating = UserRating.new(rating_params)
+    @user_rating.user = User.find(params[:user_id])
```
Comment (louisizumi) [inline app/controllers/user_ratings_controller.rb]:
```suggestion
```

Same as above for here

**Review by XR89 — APPROVED** [review]

File: app/controllers/user_ratings_controller.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+class UserRatingsController < ApplicationController
+  skip_before_action :authenticate_user!, only: %i[new]
+  def new
+    @user_rating = UserRating.new
+    @user = User.find(params[:user_id])
+    # raise
+  end
+
+  def create
+    @user_rating = UserRating.new(rating_params)
```
Comment (izumilink95) [inline app/controllers/user_ratings_controller.rb:10]:
```suggestion
```

and here will change

**Review by XR89 — APPROVED** [review]

File: app/controllers/user_ratings_controller.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
+class UserRatingsController < ApplicationController
+  skip_before_action :authenticate_user!, only: %i[new]
+  def new
+    @user_rating = UserRating.new
+    @users = User.where('session_id == current_user.session_id')
```
Comment (louisizumi) [inline app/controllers/user_ratings_controller.rb]:
```suggestion
    @users = User.where(session_id: current_user.id)
```

**Pull request merged**